core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelFilesFinder.java

```diff
@@ -98,7 +98,8 @@ private boolean isTestProjectFolder(IProject project, IResource f) {
 	 */
 	public boolean isFuseCamelContentType(IFile ifile) throws CoreException {
 		if( ifile != null
-				&& ifile.isSynchronized(IResource.DEPTH_ZERO)){
+				&& ifile.isSynchronized(IResource.DEPTH_ZERO)
+				&& ifile.isLocal(IResource.DEPTH_ZERO)){
```

Comment:
it is effectively deprecated but it is the only to avoid the stack in log as the deprecated method is still in use in Eclipse internals